src/components/radio/radio.ts

```diff
@@ -8,8 +8,9 @@ export interface RadioOptions extends BaseOptions<string> {
     type?: string,
     inline?: boolean,
     value?: string,
-    values?:Array<{}>,
-    defaultValue?: string
+    values?: Array<{}>,
+    defaultValue?: string,
+    name?: string
```

Comment:
Why does ```name``` need to be included in the settings of the component?  You are just passing the ```key``` to the name field in the template, and this is never used.